src/TurbulenceClosures/turbulence_closure_implementations/TKEBasedVerticalDiffusivities/catke_vertical_diffusivity.jl

```diff
@@ -258,33 +259,33 @@ end
     end
 end
 
-@inline function κuᶜᶜᶠ(i, j, k, grid, closure, velocities, tracers, buoyancy, surface_buoyancy_flux)
+@inline function κuᶜᶜᶠ(i, j, k, grid::AbstractGrid{FT}, closure, velocities, tracers, buoyancy, surface_buoyancy_flux) where FT
     w★ = ℑzᵃᵃᶠ(i, j, k, grid, turbulent_velocityᶜᶜᶜ, closure, tracers.e)
     ℓu = momentum_mixing_lengthᶜᶜᶠ(i, j, k, grid, closure, velocities, tracers, buoyancy, surface_buoyancy_flux)
     κu = ℓu * w★
     κu_max = closure.maximum_viscosity
-    return min(κu, κu_max)
+    return convert(FT, min(κu, κu_max))
```

Comment:
```suggestion
    FT = eltype(grid)
    return convert(FT, min(κu, κu_max))
```

(following YASGuide --- type parameters to be used only for dispatch)

Reply:
Aslo better to split this into two lines rather than convert and perform `min` on the same line.